notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -18,31 +18,61 @@
   },
```

Comment:
<p>Can you use seaborn for both of the plots? and make them a bit larger</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Changed :+1: